lib/letters.ml

```diff
@@ -1,14 +1,38 @@
-type config = {
-  sender : string;
-  username : string;
-  password : string;
-  hostname : string;
-  port : int option;
-  with_starttls : bool;
-  ca_dir : string;
-}
-
-type body = Plain of string | Html of string | Mixed of string * string * (string option)
+module Config = struct
```

Comment:
Please provide at least `equal` and `show`, ideally `pp` as well for easier debugging and testing.

Reply:
This is actually part of the other PR, not this one. I'll create separate ticket to get forward with this.

Reply:
Thanks!